src/main/java/se/sundsvall/alkt/integration/db/entity/CaseDecisionEntity.java

```diff
@@ -0,0 +1,59 @@
+package se.sundsvall.alkt.integration.db.entity;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
```

Comment:
You could replace
```
@Getter
@Setter
```
with
```
@Data
```
and remove the equals and hashcode methods (not sure if NoArgs and/or AllArgs is needed though) if you want to save some source code bytes.

Comment goes for all entities.

Reply:
It will also come with `@HashCode`, `@Equals` and `@ToString` which could cause performance issues on entities. That's why I've done it like this :)

Reply:
Performance issues? What?

Reply:
Ignore comment as entity relationships will cause stack overflow when using lomboks data annotation (which generates "faulty" tostring method).

Reply:
If Lombok does wrong, then don't use Lombok?
There are other ways...

Reply:
Let's not go down that road again 😄. Let's just agree that Lomboks data annotation is not optimal due to entity relationships, but current solution is. But yes, you are right - there is also other ways to solve it.